src/PrpShop/PRP/Surface/QMipmap.cpp

```diff
@@ -48,17 +50,39 @@ static void setExportDir(const QString& filename)
     settings.setValue("ExportDir", dir.absolutePath());
 }
 
+std::unique_ptr<unsigned char[]> getTextureData(plMipmap *tex, size_t level=0)
+{
+    if (level >= tex->getNumLevels())
+        level = tex->getNumLevels() - 1;
+
+    size_t size = tex->GetUncompressedSize(level);
+    auto imageData = std::unique_ptr<unsigned char[]>{ new unsigned char[size] };
```

Comment:
```suggestion
    auto imageData = std::make_unique<unsigned char[]>(size);
```

Reply:
I chose to do it this way to avoid the automatic value initialization, since this buffer is being created only to be immediately overwritten.  I would use `make_unique_for_overwrite` but that requires at least C++20.

At least, that's my understanding based on the specs and this answer here: https://stackoverflow.com/questions/58050872/what-does-stdmake-unique-for-overwrite-do-vis-a-vis-stdmake-unique/58050873#58050873

Reply:
The automatic value initialization will almost certainly be optimized to a single memset...  So I'd still prefer to see `make_unique` here as well

Reply:
Good to know!